docs/usage.md

```diff
@@ -0,0 +1,325 @@
+  response from the server before erroring out.
+- `--log-level`: the level of verbosity in the logs -- accepts the standard
+  Python log level names (`ERROR`, `INFO`, `DEBUG`, etc.)
+- `--ecs-log`: Log messages in Elastic Config Schema
+- `--help`: Show the command's help text
+
+All the load commands accept these flags:
+
+- `--allow-failure`: Don't error out if a resource fails to import -- just skip
+  past it.
+- `--delete-after-import`: Delete files after they've been loaded into the
+  target system, to make it easy to find ones that might have failed.
+
+Most of the dump subcommands support:
+
+- `--purge`: when dumping, delete any file that was already in the directory and
+  not affected by the operation. (Usually because the resource that file referred
+  to no longer exists in your stack.) In this mode, Stacker will display a list
+  of files it plans to delete, and prompt you for confirmation before deleting
+  them.
+- `--force-purge`: The same as `--purge`, but with no prompt for confirmation
+  before deleting.
+- `--include-managed`: Dump the "managed" resources, which are the ones created
+  by the system rather than by users. (Some resources do not have this option.)
+
+The `system-load` subcommand accepts all the `load` flags, and also:
+
+- `--retries`: Try this many times to load the resources into the target.
+- `--temp`: Copy the dump file to a temporary directory before importing.
+- `--stubborn`: Try multiple times to import things that failed, by moving the
+  dump files to a temporary directory, continuing past failures, and deleting
+  successfully imported resource files.
+
+The `system-dump` subcommand accepts all the `dump` flags, and also:
+
+- `--include-experimental`: Dump resources which cannot yet be loaded back in
+  (agent policies and package policies.)
+
+## Tips and Tricks
+
+This section describes some recommended uses for Stacker, and how to go about
+them effectively.
+
+### Using Stacker to back up your Elastic configuration
+
+Stacker dumps all the data it can handle as JSON files in a defined directory
+structure. The easiest way to store data like this is in a Git repository.
+
+Create a new repo, and in the root of the repo place a `stacker.yaml` file
+which sets `defaults.options.data_directory: "."`. Now, while you're in this
+directory, Stacker will use the configuration file in that directory first, and
+dumped files will be put directly in the repo. With this setup in place, you can
+back up your data by running `stacker system-dump`, then review the changes and
+commit them.
+
+### Using Stacker to move configuration between Elastic Stacks
+
+Stacker can be used to keep configuration in sync between several different
+Elastic Stacks (for example, a staging and a production environment.)
+
+To configure Stacker for this use case, you may want to start with
+[`stacker.example.yaml`](../stacker.example.yaml) which shows how this can work.
```

Comment:
```suggestion
[`stacker.example.yaml`](/stacker.example.yaml) which shows how this can work.
```

I think you can do this to use absolute path from the base of the repo.